src/components/Editor/Footer.js

```diff
@@ -139,11 +139,6 @@ class SourceFooter extends PureComponent {
   }
 
   renderCommands() {
-    const { selectedSource } = this.props;
-    if (!shouldShowPrettyPrint(selectedSource)) {
-      return null;
-    }
```

Comment:
how can we get rid of this? aren't there other scenarios where we don't want to pretty print?

Reply:
This check is done in two places! The check here blocks all of the other buttons (blackbox etc) from showing up if `shouldShowPrettyPrint(selectedSource)` is false. I would think a false  on`shouldShowPrettyPrint(selectedSource)` should only block the pretty print button .. right?

Reply:
the other check is on line 40